### DISCUSSION TITLE
Can `gtk-rs` be used in game engines?

### DISCUSSION THREAD

**Discussion by Raj2032** [gtk-rs/gtk4-rs#1527] [General]
For example egui can be integrated into game engines, is it possible to do the same with gtk-rs?

**Comment by sdroege** [comment]
You'll probably get more replies about this at https://discourse.gnome.org

**Comment by Raj2032** [comment]
@sdroege Does gnome actually maintain gtk4-rs?

**Reply by sdroege** [reply]
Some of the GTK maintainers are on the GNOME discourse, and generally you'll find more people there being able to answer generic GTK questions.

**Reply by Raj2032** [reply]
@sdroege But what about using gtk-rs to do programming though. I thought this would be the better place given how you guys maintain gtk-rs?

**Reply by sdroege** [reply]
There's nothing special about gtk-rs in this regard. If you can use GTK for game engines then you can use the Rust bindings for that purpose too.

**Reply by Raj2032** [reply]
Right thanks
